src/components/organisms/GlobalNavigation/GlobalNavigation.tsx

```diff
@@ -0,0 +1,67 @@
+import { Logo } from "@assets/logo";
+import { ListItem } from "@atoms/ListItem";
+import { GlobalAccount } from "../GlobalAccount";
+import { RichMenu } from "../RichMenu";
+import { useRichMenuDialog } from "./hooks";
+import type { GlobalNavigationProps } from "./type";
+
+export const GlobalNavigation = ({ items }: GlobalNavigationProps) => {
+  const { noCloseRefs, manageDialog, richMenuState } = useRichMenuDialog();
+
+  return (
+    <header className="flex items-center gap-x-6 border-b border-gray-light bg-white px-4 py-2">
+      <div>
+        <a href="/">
+          <Logo
+            height={40}
+            width={40}
+          />
+        </a>
+      </div>
+      <nav className="mr-auto">
+        <ul className="flex items-center gap-x-2">
+          {items?.map((item, index) => (
```

Comment:
Why is the optional chaining needed here? Isn't `items` always given?
```suggestion
          {items.map((item, index) => (
```

Reply:
@macfrei 
Indeeed, I'll fix it!